app/resources/base.html.template

```diff
@@ -16,101 +16,92 @@
 {% load i18n %}
 
 <!DOCTYPE html>
-{% if not env.amp %}
-<html dir="{{env.rtl|yesno:"rtl,ltr"}}" lang="{{env.lang}}">
-{% else %}
-<html amp lang="{{env.lang}}">
-{% endif %}
+
+<html {% if env.amp %}amp {% endif %}dir="{{env.rtl|yesno:"rtl,ltr"}}" lang="{{env.lang}}">
 
 <head>
-  <meta charset="utf-8">
   {% block head %}
-  {% include "gtm.html.template" %}
-  <meta name="keywords" content="{{config.keywords}}">
-  <meta http-equiv="content-type" content="text/html; charset={{env.charset}}">
-
-  {% if not env.amp %}
-  <meta name="viewport" content="width=device-width, user-scalable=yes, initial-scale=1" />
-  {% else %}
-  <meta name="viewport" content="width=device-width,minimum-scale=1">
-  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-  <script async src="https://cdn.ampproject.org/v0.js"></script>
-  {% endif %}
+    <meta charset="utf-8">
```

Comment:
Sorry I didn't notice in the previous round, but it should be charset="{{env.charset}}" instead. (It may not be UTF-8.) Also I believe you can remove line ```<meta http-equiv="content-type"...``` below instead which is redundant.

Reply:
https://www.ampproject.org/docs/reference/spec#required-markup
AMP site isn't saying meta charset="utf-8" as a mandatory, but AMP HTML validator shows error when I put <meta http-equiv="content-type"... I added if env.amp to this part too. Maybe, we'll force AMP page to utf-8 only.

Reply:
I mean, you should always just use ```<meta charset="{{env.charset}}">``` and not ```<meta http-equiv="content-type"...```, even for non-AMP page. ```<meta charset="{{env.charset}}">``` should be valid in non-AMP too.

Also, looks like you moved ```<meta charset="{{env.charset}}">``` to below ```<meta name="keywords">```, but didn't you say that it must be at the top of the `<head>` tag?

Reply:
Yes, you're right I moved it somehow to not first child. I revised it to the first child of <head> tag.
There were errors show on the AMP validater in the past, it seems now is gone.